src/test/java/Homework18.java

```diff
@@ -0,0 +1,42 @@
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+
+
+public class Homework18 extends BaseTest {
+
+@Test
+
+ public void playASongTest() throws InterruptedException {
+     login ("svetlana.edwards@testpro.io", "te$t$tudent");
+     playSong();
+     Assert.assertTrue(isSongPlaying());
+ }
+
+ public void playSong() {
+     WebElement playNextButton = driver.findelement(By.xpath("//i[@data-testid='play-next-btn']"));
+     WebElement playButton = driver.findelement(By.xpath("//span[@data-testid='play-btn']"));
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/235849126-17d7fc33-4830-45e2-ae41-15f72a36f238.png)

HINT

```
     WebElement playNextButton = driver.findElement(By.xpath("//i[@data-testid='play-next-btn']"));
     WebElement playButton = driver.findElement(By.xpath("//span[@data-testid='play-btn']"));
```

Reply:
Thank you, I updated the file